app/src/main/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityView.kt

```diff
@@ -78,6 +80,15 @@ class AllPatientsInFacilityView(
         .subscribe { downstreamUiEvents.onNext(it) }
   }
 
+  @SuppressLint("CheckResult")
+  private fun forwardListScrollEventsToDownstream(screenDestroys: Observable<ScreenDestroyed>) {
```

Comment:
I know this is not part of this commit, but instead of using a `PublishSubject` can you make streams out of these, merge them and expose it as `uiEvents` property?

Reply:
That's a good idea, I'll do this.